Directory.Packages.props

```
@@ -43,7 +43,7 @@
     <PackageVersion Include="Microsoft.TestPlatform.ObjectModel" Version="$(MicrosoftNETTestSdkVersion)" />
     <PackageVersion Include="Microsoft.TestPlatform.TranslationLayer" Version="$(MicrosoftNETTestSdkVersion)" />
     <PackageVersion Include="Microsoft.WindowsAppSDK" Version="1.0.0" />
-    <PackageVersion Include="Polyfill" Version="6.2.3" />
+    <PackageVersion Include="Polyfill" Version="6.7.0" />
```

I have just triggered sync of the latest version on our internal feed.

```suggestion
    <PackageVersion Include="Polyfill" Version="6.9.0" />
```